src/lib/map/analysis/SemiTransparentHeader.tsx

```diff
@@ -69,10 +71,32 @@ export function SemiTransparentHeader({
         />
       </div>
       <div className={semiTransparentHeader('__SampleCounter')}>
-        <p>
-          {visibleEntitiesCount || 0} of {totalEntitesCount || 0} samples
-          visible
-        </p>
+        <table>
+          <thead>
+            <tr>
+              <th title="Counts"></th>
+              <th title="Values">Samples</th>
```

Comment:
☝️ This shouldn't always be hardcoded. It should be the entity of the overlay variable. Could be "participants"

Reply:
See above :-)